Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -29,16 +29,19 @@ public class MainViewModel : BaseModel, ISavable
         #region Private Fields
 
         private const string DefaultOpenResultModifiers = "Alt";
+        private const string DefaultCycleHistoryModifiers = "Alt";
 
         private bool _isQueryRunning;
         private Query _lastQuery;
         private string _queryTextBeforeLeaveResults;
 
+
         private readonly FlowLauncherJsonStorage<History> _historyItemsStorage;
         private readonly FlowLauncherJsonStorage<UserSelectedRecord> _userSelectedRecordStorage;
         private readonly FlowLauncherJsonStorage<TopMostRecord> _topMostRecordStorage;
         internal readonly Settings _settings;
         private readonly History _history;
+        private int _lastHistoryIndex = 1;
```

Comment:
this doesnt need to be underscore, code base may have some private variables with underscores but new code should use just lowercase and we will gradually rename these underscores. the standard naming practice for private variables is no longer underscore.